contracts/teller-network/MetadataStore.sol

```diff
@@ -187,6 +192,22 @@ contract MetadataStore is Ownable {
         emit OfferUpdated(msg.sender, _offerId, _asset, _statusContactCode, _location, _currency, _username, _paymentMethods, _marketType, _margin);
     }
 
+    /**
+    * @notice Remove user offer
+    * @param _offerId Id of the offer to remove
+    */
+    function removeOffer(
+        uint256 _offerId
+    ) public {
+        require(userWhitelist[msg.sender], "User does not exist");
+        require(offerWhitelist[msg.sender][_offerId], "Offer does not exist");
+        require(offers[_offerId].status == OfferStatus.Open, "Only open offers can be removed");
```

Comment:
Is the state machine really usefull? if not then we can remove it

Reply:
I'll remove it. We could always introduce it again in the future if necessary.